assets/queries/k8s/rotate_kubelet_server_certificate_not_active/query.rego

```diff
@@ -3,19 +3,23 @@ package Cx
 import data.generic.common as common_lib
 import data.generic.k8s as k8sLib
 
+commandList = {"kubelet", "kube-controller-manager"}
+
 CxPolicy[result] {
 	resource := input.document[i]
+	metadata := resource.metadata
 	specInfo := k8sLib.getSpecInfo(resource)
 	types := {"initContainers", "containers"}
 	container := specInfo.spec[types[x]][j]
-	command := "kubelet"
+	command := commandList[_]
 
 	common_lib.inArray(container.command, command)
-	k8sLib.hasFlag(container, "--feature-gates=RotateKubeletServerCertificate=false")
+	k8sLib.startWithFlag(container,"--feature-gates=")
+	contains_feature(container, "RotateKubeletServerCertificate=false")
```

Comment:
The `contains_feature` function does the same thing as the `k8sLib.hasFlag` function, could you replace it?
```suggestion
	k8sLib.hasFlag(container, "RotateKubeletServerCertificate=false")
```